documentation/plain_rules.md

```diff
@@ -566,6 +573,30 @@ Note that using NGSIv2 the BloodPressure attribute is a Number and therefore it
 }
 ```
 
+```json
+{
+    "name": "murule",
+    "text": "select *,'myrule' as ruleName from pattern [every ev=iotEvent(type='SensorMetter')]",
+    "action": {
+        "type": "update",
+        "parameters": {
+            "filter": {
+                "type": "SensorMetter",
+                "iterateAll": true
```

Comment:
Example is correct? As far as I understand by https://github.com/telefonicaid/perseo-fe/pull/492#discussion_r576127504 at the end iterateAll is not going to be used because the paginated approach will be used.

Reply:
In fact, maybe the usage of `options` could be illustrated (although looking to https://conwetlab.github.io/ngsijs/stable/NGSI.Connection.html#.%22v2.listEntities%22__anchor I'm not sure which one could make sense in Perseo update action use cases).

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/492/commits/5615a33dadf19fa18a37e38ce852733368084bc0